.github/workflows/stale.yaml

```diff
@@ -0,0 +1,34 @@
+name: 'Close stale'
+
+on:
+  schedule:
+    - cron: '0 7 * * *'
+
+permissions:
+  contents: read
+
+jobs:
+  stale:
+    # These are the permissions recommended by github.
+    permissions:
+      pull-requests: write
```

Comment:
```suggestion
      contents: read
```

and set octo-sts

Reply:
Maybe i am wrong but I thought to close PRs, you need write permissions.